bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -49,6 +49,12 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     Returns:
         dict: Dict with column names as keys and 1D numpy arrays with content as values
     """
+    # Sanitize input arguments
+    if not columns == None and isinstance(columns, str):
```

Comment:
Do you strictly need to compare with `None` first?

Reply:
uhm no, thanks :see_no_evil:

Reply:
Fixed!